src/encoded/static/scss/encoded/modules/_search.scss

```diff
@@ -370,6 +370,40 @@ $form-bg: #f3f3f3;
     display: inline-block;
 }
 
+// View control report button
+.view-type-report {
+    > button {
+        @extend .btn-info;
+        @extend .btn-sm;
+    }
+
+    .dropdown-button__content {
+        width: auto;
+        white-space: nowrap;
+
+        a {
+            font-family: Quicksand, sans-serif;
```

Comment:
I think you could use $button-font here, just in case we one day wanted to change all the instances of Quicksand - unlikely now but in theoretically possible.